web/src/lib/i18n/en.json

```diff
@@ -317,6 +317,7 @@
   "archived": "Archived",
   "asset_offline": "Asset offline",
   "assets": "Assets",
+  "assets_moved_to_trash": "Moved {count, plural, one {# asset} other {# assets}} to trash",
```

Comment:
Does `#` work similar to `{count, number}`?

Reply:
Yes exactly, we're already using the shorter `#` syntax for some of our existing translations

Reply:
Ah ok, that's cool (and easier)! :D